src/sentry/rules/filters/issue_occurrences.py

```diff
@@ -26,3 +29,26 @@ def passes(self, event: GroupEvent, state: EventState) -> bool:
         # around this by including pending updates from buffers to improve accuracy.
         issue_occurrences: int = event.group.times_seen_with_pending
         return issue_occurrences >= value
+
+    def passes_activity(self, condition_activity: ConditionActivity) -> bool:
+        try:
+            value = int(self.get_option("value"))
+        except (TypeError, ValueError):
+            return False
+
+        try:
+            group = Group.objects.get_from_cache(id=condition_activity.group_id)
+        except Group.DoesNotExist:
+            return False
+
+        now = timezone.now()
+        if now == group.first_seen:
```

Comment:
when does this evaluate to true? or is this a hack for the fake event alerts?

Reply:
yeah I had this for tests, and just to be safe